src/audio/pipeline.c

```diff
@@ -63,6 +63,7 @@ void pipeline_posn_init(struct sof *sof)
 }
```

Comment:
typo in commit message s/peline/pipeline.

Reply:
fixed